onadata/libs/utils/csv_builder.py

```diff
@@ -552,18 +557,9 @@ def _update_columns_from_data(self, cursor):
         image_xpaths = [] if not self.include_images \
             else self.dd.get_media_survey_xpaths()
```

Comment:
Seems this variable is stored twice in the `_update_ordered_columns_from_data` & `_format_for_dataframe` functions could we perhaps store them in an object attribute like `self.image_xpaths` and set it only once ? So that we avoid the duplicate call to `get_media_survey_xpaths`

Or set it in the `export_to` function? _Might save us a bit more time for some forms; We won't need to traverse the elements to figure out all the media survey xpaths twice_

Reply:
Made this change. Should be good now